public/app/features/provisioning/ConfigForm.tsx

```diff
@@ -223,8 +223,8 @@ export function ConfigForm({ data }: ConfigFormProps) {
         />
       )}
 
-      <ControlledCollapse label="Sync Settings" isOpen={false}>
-        <Field label={'Enabled'} description={'Once sync is enabled, the target cannot be changed.'}>
+      <ControlledCollapse label="Automatic Pull Settings" isOpen={false}>
```

Comment:
```suggestion
      <ControlledCollapse label="Automatic pull settings" isOpen={false}>
```
We don't really use title case in Grafana UI, this goes for field labels, button text and headings among others.